data/core/step1_get_started.md

```diff
@@ -25,11 +25,36 @@
     - utter_getstarted_new
     - utter_built_bot_before
 * deny
-    - utter_explain_stack
-    - utter_stack_details
-    - utter_explain_nlucore
+    - utter_explain_rasa_components
+    - utter_rasa_components_details
+    - utter_ask_explain_nlucorex
+* affirm OR how_to_get_started{"product":"all"}
+    - utter_explain_nlu
+    - utter_explain_core
+    - utter_explain_x
+    - utter_direct_to_step2
+
+## new to rasa + built a bot before
+* how_to_get_started
+    - utter_getstarted
+    - utter_first_bot_with_rasa
+* affirm OR how_to_get_started{"user_type": "new"}
+    - action_set_onboarding
+    - slot{"onboarding": true}
+    - utter_built_bot_before
+* affirm
+    - utter_ask_migration
+* deny
+    - utter_explain_rasa_components
+    - utter_rasa_components_details
+    - utter_ask_explain_nlucorex
+* affirm OR how_to_get_started{"product":"all"}
+    - utter_explain_nlu
+    - utter_explain_core
+    - utter_explain_x
+    - utter_direct_to_step2
 
-## new to rasa + not new to chatbots + not migrating
+## new to rasa + not new to chatbots + migrating from 
```

Comment:
this might be one of those weird git diff things that happens with stories, but this user isn't migrating